caiman/cluster.py

```diff
@@ -189,22 +189,25 @@ def setup_cluster(backend:str = 'multiprocessing',
     """
     Setup and/or restart a parallel cluster.
     Args:
-        backend: str
-            'multiprocessing' [alias 'local'], 'ipyparallel', and 'SLURM'
-            ipyparallel and SLURM backends try to restart if cluster running.
-            backend='multiprocessing' raises an exception if a cluster is running.
-        n_processes: int
-            Set number of processes to use. If None, is automatically calculated 
-        single_thread: bool
-            If True, do not use cluster
-        ignore_preexisting: bool
+        backend
+            One of:
+                'multiprocessing' - Use multiprocessing library
+                'ipyparallel' - Use ipyparallel instead (better on Windows?)
```

Comment:
interesting, is this related to the requirement to have an `if __name__ == "__main__"` on windows?